main.nf

```diff
@@ -24,10 +24,16 @@ process PREPROCESS {
   container 'docker.io/satijalab/seurat:5.0.0'
 
   input:
-      path data 
+      tuple val(sample), path(data) 
```

Comment:
Is `sample` the sample name from the samplesheet.csv? How do we use it? Does the path() return the path of the specific sample?